src/jsonSchema.ts

```diff
@@ -1,8 +1,8 @@
 import * as Ajv from 'ajv';
 import { appendErrors, Resolver, transformToNestObject } from 'react-hook-form';
-import type { ErrorObject, Options, ValidateFunction } from 'ajv';
-import type { JSONSchema4, JSONSchema6, JSONSchema7 } from 'json-schema';
-import type {
+import { ErrorObject, Options, ValidateFunction } from 'ajv';
```

Comment:
Why did you delete the "type"? It was correct

Reply:
<img width="397" alt="Screen Shot 2020-09-18 at 4 25 07 pm" src="https://user-images.githubusercontent.com/10513364/93563574-85ff4d80-f9cb-11ea-9910-c5c2a70a4615.png">